.github/workflows/workflow.yml

```diff
@@ -22,7 +22,7 @@ jobs:
   test:
     name: "Unit test"
     runs-on: ubuntu-latest
-    timeout-minutes: 5
+    timeout-minutes: 10
```

Comment:
hm, did you notice an increase in timeouts?

Reply:
yes, i added `-race -bench` to align with GK which increased the timeouts

Reply:
the reason we added is to make sure to run benchmarks often so we don't break them